src/main/java/frc/robot/competition/CompetitionRobotContainer.java

```diff
@@ -276,9 +280,26 @@ private void configureBindings() {
     // .whileTrue(m_Wrist.setToTarget(19).alongWith(m_Elevator.setToTarget(13.9)))
     // .onFalse(m_Wrist.stow());
 
+    m_manipController
+        .x()
+        .whileTrue(
+            new SequentialCommandGroup(
+                new VarShootPrime(
+                    m_Wrist,
+                    m_Shooter,
+                    m_Elevator,
+                    m_poseEstimator,
+                    RobotConstants.SHOOT_POINT,
+                    RobotConstants.VELOCITY_RANGE,
+                    RobotConstants.DISTANCE_RANGE,
+                    RobotConstants.HEIGHT_LENGTH_COEFF,
+                    RobotConstants.SHOOTER_RPM_TO_MPS),
+                m_Wrist.setToTargetCmd(RobotConstants.WRIST_HIGH_LIM),
+                m_Shooter.setSpeed(0)));
```

Comment:
I lied. You'll wanna split this into `whileTrue` and `onFalse` since the setToTargetCmd will never run, as VarShootPrime will never finish

Reply:
Also, if this works with the shooter only at a single speed, then I would remove the shooter from this class for now, and have the trigger for this command be whether or not the note is in the shooter or not. then as we drive the wrist will auto adjust, and Cam can rev up the shooter and shooter like he has been.

Perhaps we need a trigger that changes the default command of the wrist to track the target when there is a note. If we go for the amp and then don't go for the amp, we'll want to track the target, but that would require starting a new command. Something to think about